src/Order.php

```diff
@@ -133,7 +134,7 @@ public function toJson(): array
      * @param Order $orderToCompare
```

Comment:
@ampdan get rid of those phpdocs code is already type hinted so that is just noisy. you an leave the @var ones to the attributes.